Wikipedia/Code/AutoLayoutSafeMultiLineButton.swift

```diff
@@ -0,0 +1,49 @@
+
+// See multi-line button thread: https://stackoverflow.com/q/23845982/135557
+@IBDesignable
+class AutoLayoutSafeMultiLineButton: UIButton {
+    func multiLineSafeSetup () {
```

Comment:
Could this be abstracted into a `SetupButton` class similar to `SetupView` so it's consistent & reusable?

Reply:
fixed!